<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KFK9388M" failures="1" tests="9" name="com.TestNgExtentReport.Example" time="40.513" errors="1" timestamp="2023-01-27T21:51:27 IST" skipped="0">
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="4.601"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: Lenovo
Entered password: 
It display correct Alert message]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="5.056"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: 
Entered password: Ideapad
It display correct is error message]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="4.803"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: Asus
Entered password: 1234
Error message is displayed: true
Error message is displayed as: Invalid Login
It displayed correct message: True]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="flush" time="4.897">
    <error message="Failed to open file:/C:/Users/91766/eclipse-workspace/Extension%20Report/reporter.html. Error message: The system cannot find the file specified.
" type="java.io.IOException">
      <![CDATA[java.io.IOException: Failed to open file:/C:/Users/91766/eclipse-workspace/Extension%20Report/reporter.html. Error message: The system cannot find the file specified.

at java.desktop/sun.awt.windows.WDesktopPeer.ShellExecute(WDesktopPeer.java:115)
at java.desktop/sun.awt.windows.WDesktopPeer.browse(WDesktopPeer.java:101)
at java.desktop/java.awt.Desktop.browse(Desktop.java:533)
at com.TestNgExtentReport.BaseClass.flush(BaseClass.java:130)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:400)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- flush -->
  <system-out/>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="4.572"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: Hello
Entered password: Dream
Error message is displayed: true
Error message is displayed as: Invalid Login
It displayed correct message: True]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="5.837"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: hELLO
Entered password: dREAMS
Error message is displayed: true
Error message is displayed as: Invalid Login
It displayed correct message: True]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="5.968"/>
  <system-out>
    <![CDATA[***positive testing***
Entered username: Hello
Entered password: Dreams
Login is successful]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="orangeHRMLogin" time="4.686"/>
  <system-out>
    <![CDATA[***Neagtive testing***
Entered username: username
Entered password: Dreams
Error message is displayed: true
Error message is displayed as: Invalid Login
It displayed correct message: True]]>
  </system-out>
  <testcase classname="com.TestNgExtentReport.Example" name="loginPageLink" time="0.093">
    <failure message="Title missmatched expected [OrangeHRM1] but found [OrangeHRM]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Title missmatched expected [OrangeHRM1] but found [OrangeHRM]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.TestNgExtentReport.Example.loginPageLink(Example.java:127)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginPageLink -->
  <system-out>
    <![CDATA[Link Name: OrangeHRM
wrong Link name is displayed: OrangeHRM]]>
  </system-out>
</testsuite> <!-- com.TestNgExtentReport.Example -->
